apps/openmw/mwstate/quicksavemanager.cpp

```diff
@@ -0,0 +1,73 @@
+#include "quicksavemanager.hpp"
+
+#include <sstream>
+
+MWState::QuickSaveManager::QuickSaveManager(std::string &saveName, int maxSaves){
+    this->saveName = saveName;
```

Comment:
Let's use the oreject codestyle (use [this](http://wiki.openmw.org/index.php?title=Policies_and_Standards) guide): 
1. Fields should have the "m" prefix.
2. "{" should be on the separate line.
3. Use headers grouping.

Reply:
http://wiki.openmw.org/index.php?title=Code_Formatting_Conventions#Includes